userspace/falco/configuration.cpp

```diff
@@ -32,9 +32,10 @@ limitations under the License.
 
 #include "configuration.h"
 #include "logger.h"
-
+#include <iterator>
 #include <re2/re2.h>
-
+#include "valijson/adapters/yaml_cpp_adapters.hpp"
+#include "ValidationUtils.h"
```

Comment:
Where does this come from? :thinking:

Reply:
Hey @leogr , could you please take a look at this PR so that we can finish our work on it?